config.toml.example

```diff
@@ -418,9 +430,10 @@ changelog-seen = 2
 #debug = false
 
 # Whether to download the stage 1 and 2 compilers from CI.
-# This is mostly useful for tools; if you have changes to `compiler/` they will be ignored.
+# This is mostly useful for tools; if you have changes to `compiler/` or `library/` they will be ignored.
 #
-# You can set this to "if-unchanged" to only download if `compiler/` has not been modified.
+# Set this to "if-unchanged" to only download if the compiler and standard library have not been modified.
+# Set this to `true` to download unconditionally (useful if e.g. you are only changing doc-comments).
 #download-rustc = false
```

Comment:
Also, does the test suite passes with this option?

Reply:
All but two of the UI tests pass with this enabled. I opened https://github.com/rust-lang/rust/issues/108767 to track the two failing tests.

About half the debuginfo tests fail because of https://github.com/rust-lang/simpleinfra/issues/254, I haven't bothered to get them working but I don't think it's related to this option.

The standard library unit tests fail: https://github.com/rust-lang/rust/issues/108768

`tests/run-make/translation` fails, but I'm not sure if it's related to this change or not:
```
error: failed to load fluent bundle: failed to add resource: Attempt to override an existing message: "parse_struct_literal_body_without_path".
```